gm4_intro_song/data/gm4_intro_song/functions/init.mcfunction

```diff
@@ -0,0 +1,12 @@
+scoreboard objectives add gm4_intro_song dummy
+scoreboard objectives add gm4_intro_song_t dummy
+
+scoreboard objectives add gm4_intro_walked minecraft.custom:walk_one_cm
+
+# enable intro song by default, upon first installation
+execute unless score $enabled gm4_intro_song = $enabled gm4_intro_song run scoreboard players set $enabled gm4_intro_song 1
```

Comment:
If this is a separate module an enabled config may not be necessary? Except maybe for when you want to temporarily disable/enable it on a server?

Reply:
I just left in the option to disable it while still having it installed.   I can take this out if you think it's unnecessary since it's standalone